report-api/src/main/resources/template/affiliation.sparql

```diff
@@ -13,18 +13,20 @@ SELECT DISTINCT
     ?facultyId
     ?departmentId
     ?groupId
+    ?modifiedDate
 WHERE {
   GRAPH ?g {
     ?uri :identifier ?publicationIdentifier ;
          :status ?publicationStatus ;
-         :modifiedDate ?modifiedDate ;
+         :modifiedDate ?modifiedDateRaw ;
          :contributor ?contributor .
 
-    FILTER(?modifiedDate >= "__AFTER__"^^xsd:dateTime)
-    FILTER(?modifiedDate < "__BEFORE__"^^xsd:dateTime)
+    FILTER(?modifiedDateRaw >= "__AFTER__"^^xsd:dateTime)
+    FILTER(?modifiedDateRaw < "__BEFORE__"^^xsd:dateTime)
```

Comment:
Necessary? here as elsewere?

Reply:
Yes, because it is binded as String when selecting